test/native/pom.xml

```diff
@@ -190,6 +190,17 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-mapreduce-client-core</artifactId>
+            <version>${hadoop.version}</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>*</groupId>
+                    <artifactId>*</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
```

Comment:
- There is definitely a big bug with the HiveServer2 JDBC Driver's handling of this. I've opened https://issues.apache.org/jira/browse/HIVE-28652 .